cmd/composer-cli/blueprints/save.go

```diff
@@ -133,8 +133,8 @@ func saveBlueprint(data, commit, path string) error {
 		filename = filename + "-" + commit
 	}
 	filename = filepath.Base(filename + ".toml")
-	if filename == "/" || filename == "." || filename == ".." {
-		return fmt.Errorf("ERROR: Invalid blueprint filename: %s", name)
+	if filename == ".toml" {
```

Comment:
Why this change? I assume it's because `filepath.Base` cannot return `/`, `.` and `..`? Might be worth documenting in the commit message.

Reply:
Right, since Base is being used it really can only end up with an empty filename with the mandatory .toml extension which means it was empty, which really should never happen since all blueprints need a name.